pkg/client/tests/fake_client_test.go

```diff
@@ -132,7 +132,7 @@ func TestFakeClientsetInheritsNamespace(t *testing.T) {
 	}
 
 	_, err = tc.CoreV1().Pods("nsA").Create(testPod("", "pod-1"))
-	if err == nil {
+	if err != nil {
```

Comment:
The logic here is correct and does not need to be modified. If error message logged out, the unit test will fail.